pkg/cloud/cloud_test.go

```diff
@@ -240,12 +240,14 @@ func checkVolumes(t *testing.T, podSpec corev1.PodSpec) {
 func checkContainerCommand(t *testing.T, podSpec corev1.PodSpec) {
 	binBash := "/bin/bash"
 	dashC := "-c"
+	// This script should be present on every node.
```

Comment:
```suggestion
	// This script should be present on every master node.
```

Reply:
I wonder if it matters that masters and workers will communicate with API Server using different load balancers. Should it be documented somewhere? That if worker is running CNM replica (Azure) then it is connected to a different endpoint? @JoelSpeed It is mostly an implementational detail, but still.

Reply:
We will need this to be every node for Azure, so we are fixing that on the MCO side already. This comment will be updated to reflect the changes and the differences once that is merged